packages/gatsby-image/src/index.js

```diff
@@ -313,8 +376,9 @@ class Image extends React.Component {
       className: placeholderClassName,
     }
 
-    if (fluid) {
-      const image = fluid
+    if (fluid || fluidImages) {
+      // Image variants support. First element to `image`, remaining to `imageVariants`
+      const [image, ...imageVariants] = fluidImages ? fluidImages : [fluid]
```

Comment:
I wonder if we should do the following to reduce some of the code below.
```
const imageVariants = fluidImages ? fluidImages : [fluid]
const image = imageVariants[0]
```

Reply:
Sorry, how is this reducing the code? You've just split the single line into two lines, this does exactly the same thing:

```JS
const [image, ...imageVariants] = fluidImages ? fluidImages : [fluid]
```

It either takes an array of `fluidImages` or puts the single `fluid` item into an array. The first element of the array is destructured into the const `image`, and the rest into the const `imageVariants`. The comment directly above it should make that clear?

Reply:
Oh nvm, you wanted to keep the original `fluidImages` array but mapped to `imageVariants`.

It was like that originally, but was specially handling the first element(see the [outdated code here](https://github.com/gatsbyjs/gatsby/blob/46dc7318d291b299bd79b3d94bf3542e5ab5612b/packages/gatsby-image/src/index.js#L79)), this may no longer be an issue as pointed in a [response to one of your other review comments](https://github.com/gatsbyjs/gatsby/pull/13395#discussion_r291798702).